quarkus-extension/quarkus-app/pom.xml

```diff
@@ -23,12 +28,11 @@
         <dependency>
             <groupId>com.baeldung.quarkus.liquibase</groupId>
             <artifactId>quarkus-liquibase-runtime</artifactId>
-            <version>1.0-SNAPSHOT</version>
+            <version>${project.version}</version>
```

Comment:
is this property defined already?

Reply:
yes, it will automatically redirect to its parent extension `quarkus-extension` and will fetch version from there